LayoutTests/svg/dynamic-updates/SVGFEComponentTransferElement-dom-amplitude-attr.html

```diff
@@ -66,21 +66,21 @@ <h1>SVG 1.1 dynamic update tests</h1>
 rootSVGElement.setAttribute("width", "400");
 rootSVGElement.setAttribute("height", "200");
 
-shouldBeEqualToString("feRFunc.getAttribute('amplitude')", "3");
-shouldBeEqualToString("feGFunc.getAttribute('amplitude')", "3");
+shouldBeEqualToString("feRFunc.getAttribute('amplitude')", "1");
+shouldBeEqualToString("feGFunc.getAttribute('amplitude')", "2");
 shouldBeEqualToString("feBFunc.getAttribute('amplitude')", "3");
-shouldBeEqualToString("feAFunc.getAttribute('amplitude')", "3");
+shouldBeEqualToString("feAFunc.getAttribute('amplitude')", "4");
 
 function repaintTest() {
-    feRFunc.setAttribute("amplitude", "1");
-	feGFunc.setAttribute("amplitude", "1");
-	feBFunc.setAttribute("amplitude", "1");
-	feAFunc.setAttribute("amplitude", "1");
+    feRFunc.setAttribute("amplitude", "5");
+	feGFunc.setAttribute("amplitude", "6");
```

Comment:
I would have used the opportunity to fix the trailing white space as well.

Reply:
Done.